sources/ElkArte/Modules/Random/Display.php

```diff
@@ -44,6 +45,7 @@ public static function hooks(EventManager $eventsManager)
 
 			add_integration_function('integrate_topic_query', '\\ElkArte\\Modules\\Random\\Display::followup_topic_query', '', false);
 			add_integration_function('integrate_display_message_list', '\\ElkArte\\Modules\\Random\\Display::followup_message_list', '', false);
+			add_integration_function('integrate_display_message_list', '\\ElkArte\\Modules\\Random\\Display::load_likes', '', false);
```

Comment:
Where that is located is contingent on `if (empty($modSettings['enableFollowup']))` which I think means followups have to be disabled for `load_likes` to be called.  I thing that should be moved to its onw likes_enabled check ?